.github/workflows/test.yml

```diff
@@ -16,24 +16,24 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python: "3.11"
-            image_tag: "nightly"
+          - image_tag: "nightly"
             pulp_api_root: "/relocated/djnd/"
-          - python: "3.8"
-            image_tag: "3.28"
+            python: "3.11"
+          - image_tag: "3.28"
             lower_bounds: true
-          - python: "3.12"
-            image_tag: "3.26"
-          - python: "3.8"
-            image_tag: "3.25"
-          - python: "3.9"
-            image_tag: "3.24"
+            python: "3.8"
+          - image_tag: "3.26"
+            python: "3.12"
+          - image_tag: "3.25"
+            python: "3.8"
+          - image_tag: "3.24"
             pulp_api_root: "/relocated/djnd/"
-          - python: "3.10"
-            image_tag: "3.35"
-          - python: "3.11"
-            image_tag: "latest"
+            python: "3.9"
+          - image_tag: "3.35"
+            python: "3.10"
+          - image_tag: "latest"
             lower_bounds: true
+            python: "3.11"
```

Comment:
Ah right dictionaries sorted by keys for stability...
(should be the same as before otherwise.)